<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepProgressBar</class>
 <widget class="QWidget" name="StepProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
	background: transparent;
	border: none;
}

QPushButton {
	background: #F0F0F0;
	border: none;
	border-radius: 3px;
}

QLabel {
	font-family: Open Sans;
	font-size: 18px;
	line-height: 0px;
	letter-spacing: 0%;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="step1">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/step1_ready.svg</normaloff>:/images/Project/step1_ready.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="line1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="step2">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/step2_upcoming.svg</normaloff>:/images/Project/step2_upcoming.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="line2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="step3">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/step3_upcoming.svg</normaloff>:/images/Project/step3_upcoming.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="line3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="step4">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/step4_upcoming.svg</normaloff>:/images/Project/step4_upcoming.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="line4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="step5">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/step5_upcoming.svg</normaloff>:/images/Project/step5_upcoming.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="line5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>6</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="step6">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/Project/step6_upcoming.svg</normaloff>:/images/Project/step6_upcoming.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>46</width>
          <height>46</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="labelContainer" native="true">
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Basic Info</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>203</x>
        <y>10</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Additional Fileds</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>405</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Directories</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Database and Configuration Management</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>806</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Integration</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label6">
      <property name="geometry">
       <rect>
        <x>999</x>
        <y>10</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>AI Connection</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
